server/models/user.js

```diff
@@ -1,76 +1,41 @@
 import mongoose from 'mongoose';
-import httpStatus from 'http-status';
-import APIError from '../helpers/APIError';
+import EscalationPolicySchema from './escalationPolicy';
+
+const emailRegex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/; // eslint-disable-line
 
-/**
- * User Schema
- */
 const UserSchema = new mongoose.Schema({
-  username: {
+  name: {
     type: String,
-    required: true
+    required: true,
   },
-  mobileNumber: {
+  email: {
     type: String,
+    validate: {
+      validator: v => emailRegex.test(v),
+      message: '{VALUE} is not a valid email address'
+    },
     required: true,
-    match: [/^[1-9][0-9]{9}$/, 'The value of path {PATH} ({VALUE}) is not a valid mobile number.']
+  },
+  auth: String,
+  escalationPolicy: {
+    type: EscalationPolicySchema,
+    default: null
+  },
+  groups: [mongoose.Schema.Types.ObjectId],
+  devices: [{
+    type: String,
+    address: String,
+    id: String,
+    name: String
+  }],
+  role: {
+    type: Number, // This can be an enum, should we make this an enum?
```

Comment:
I think this is something we can discuss in a planning iteration in the future, but I'll leave this here so that there's an understanding that at the moment, this is ambiguous.